mchf-eclipse/drivers/audio/audio_nr.h

```diff
@@ -28,26 +28,35 @@ typedef struct __MCHF_SPECIALMEM NoiseReduction
 	float32_t 					last_sample_buffer_L [NR_FFT_L / 2];
```

Comment:
__MCHF_SPECIALMEM does not belong to a type definition (as the compiler told you).
A type in C does not have a memory location.
See below for next comment.